app/forms/idv/phone_form.rb

```diff
@@ -46,14 +48,20 @@ def user_has_multiple_phones?
 
     attr_writer :phone, :otp_delivery_preference
 
-    def initial_phone_value(input_phone)
-      initial_phone = input_phone
-      initial_phone ||= begin
+    def initial_value(input_phone)
```

Comment:
I feel like tuple returns are uncommon, I'd document this to help future us. Maybe even rename to a plural?

```suggestion
    # @return [nil, Array(String, String)] a tuple of (country_code, formatted phone) or nil
    def initial_values(input_phone)
    def initial_values(input_phone)
```